noggin/form/password_reset.py

```diff
@@ -19,8 +19,10 @@ class NewPasswordForm(BaseForm):
 
     password_confirm = PasswordField(_('Confirm New Password'))
 
-    otp = StringField(
-        _('OTP Token'), description=_("Enter your OTP token if you have enrolled one")
+    otp = PasswordField(
```

Comment:
I think this can remain a StringField, the value is temporary so no need to hide it.